Quark_Sketch/src/arduino/libraries/SPI_LIB.h

```diff
@@ -1,15 +1,17 @@
 /*
```

Comment:
@calvinatintel yes this was pretty much written from scratch only looking at the old GPL version to ensure that the API is compatible